.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
               export PATH=$HOME/bin:$PATH
               cf login -a https://api.fr.cloud.gov -u $DEV_USER -p $DEV_PASSWORD -o $ORG_NAME -s $SPACE_NAME
               echo "Login successful."
-              cf zero-downtime-push opre-ops-test -f manifest.yml
+              cf push opre-ops-test -f manifest.yml
```

Comment:
[question]: Why is `zero-downtime-push` being removed?

Reply:
To make deployment from local as similar as possible to CircleCI, and to reduce memory usage. I didn't confirm that the change is currently needed for a successful deployment, but I thought if we are running into memory problems it's good to start trimming down unnecessary things.

Reply:
Update: at your request have have confirmed that `zero-downtime-push` blocks deployment by exceeding the memory limit. So I have restored `push` in the config file, and I have also removed the [autopilot](https://github.com/contraband/autopilot) plugin, which was [deprecated years ago](https://engineering.18f.gov/continuous-deployment/).

Reply:
Okay thank you for checking, it sounds like it is necessary to remove it.